Dockerfile

```diff
@@ -16,8 +17,10 @@ RUN pip install pip --upgrade
 # Install project packages
 RUN pip install -r requirements.txt
 RUN pip install -r test_requirements.txt
+RUN pip install -r doc_requirements.txt
 RUN pip3 install -r requirements.txt
 RUN pip3 install -r test_requirements.txt
+RUN pip3 install -r doc_requirements.txt
```

Comment:
We just need to choose one or the other, no reason to slow the build for testing in both python languages.  I think readthedocs uses 2, so we can just do it there.